openslides/motions/models.py

```diff
@@ -339,7 +339,11 @@ def get_new_version(self, **kwargs):
         attributes, it is populated with the data from the last version
         object if such object exists.
         """
-        new_version = MotionVersion(motion=self, **kwargs)
+        if self.pk is None:
+            # Do not reference the MotionVersion object to an unsaved motion
```

Comment:
This looks strange: Where do we ever try to retrieve a new version for an unsaved motion? See line 154 ans 184 in this file.

If we ever do so, how can we ensure that the reference is set sometime?